lib/cisco_node_utils/feature.rb

```diff
@@ -80,7 +80,12 @@ def self.nv_overlay_enabled?
 
     # ---------------------------
     def self.nv_overlay_evpn_enable
-      return if nv_overlay_evpn_enabled?
+      begin
+        return if nv_overlay_evpn_enabled? && fabric_forwarding_enabled?
+        config_set('feature', 'fabric_forwarding')
+        config_set('feature', 'nv_overlay_evpn')
+      end
+    rescue
```

Comment:
Please add a comment in this rescue explaining why we got here, ie: 

```
      # The feature fabric-forwarding cli is required in some older nxos images but is not 
      # present in newer images because nv_overlay_evpn handles both features; therefore
      # feature fabric-forwarding is best-effort and ignored on cli failure.